configurations/dockerYamlTemplate.go

```diff
@@ -10,7 +10,7 @@ import (
 )
 
 type dockerfileDataYaml struct {
-	Stages map[string]stage `yaml:"stages"`
+	ServerConfig stage `yaml:"serverConfig"`
```

Comment:
Semantics do not line up.
The member name is `ServerConfig` while the data type says that it is a `stage`. Names should be semantically correct.

Reply:
Yes. Renamed it